src/modules/search/SearchModule.ts

```diff
@@ -786,16 +786,16 @@ export default class SearchModule implements ReaderModule {
   }
 
   paginate(items: Array<any>, page: number, per_page: number) {
-    var page = page || 1,
-      per_page = per_page || 10,
-      offset = (page - 1) * per_page,
-      paginatedItems = items.slice(offset).slice(0, per_page),
-      total_pages = Math.ceil(items.length / per_page);
+    let _page = page || 1,
+      _per_page = per_page || 10,
+      offset = (_page - 1) * _per_page,
+      paginatedItems = items.slice(offset).slice(0, _per_page),
+      total_pages = Math.ceil(items.length / _per_page);
```

Comment:
Just curious why you added the underscore in the name? I usually only do that for variables that will be unused

Reply:
I didn't come up with a different name :) this can be anything really . But the actions where marking how it was before as being reassigned . And not valid .

Reply:
I see yea I didn't notice that there was `_per_page` and `per_page` actually. Makes more sense now